package.json

```diff
@@ -16,14 +16,15 @@
   "scripts": {
     "test-node": "mocha --recursive -R dot test/",
     "test-dev": "npm run test-node -- --watch -R min",
-    "test-headless": "mochify --recursive -R dot --grep WebWorker --invert test/",
+    "test-headless": "mochify --recursive -R dot --grep WebWorker --invert --plugin [proxyquire-universal] test/",
     "test-coverage": "mochify --recursive -R dot --grep WebWorker --invert --plugin [ mochify-istanbul --exclude '**/+(test|node_modules)/**/*' --report text --report lcovonly --dir ./coverage ] test/",
     "test-cloud": "npm run test-headless -- --wd",
     "test-webworker": "browserify --no-commondir --full-paths -p [ mocaccino -R spec --color ] test/webworker/webworker-support-assessment.js | phantomic --web-security false",
-    "test": "npm run test-node && npm run test-headless && npm run test-webworker",
+    "test": "run-p test-node test-headless && run-s build test-webworker",
```

Comment:
Could absolutely have been an issue, but the startup time of the phantom stuff actually makes it sequential by chance. You are probably right that it doesn't make sense to save 1.5 seconds in risk of having garbled output. I usually parellize tasks when having interactive/watched builds, where I don't really care all that much.